docs/domains/sparse_linear_algebra.rst

```diff
@@ -38,3 +38,145 @@ Currently known limitations:
   ``oneapi::mkl::unimplemented`` exception.
 - Scalar parameters ``alpha`` and ``beta`` should be host pointers to prevent
   synchronizations and copies to the host.
+
+
+cuSPARSE backend
+----------------
+
+Currently known limitations:
+
+- The COO format requires the indices to be sorted by row. See the `cuSPARSE
+  documentation
+  <https://docs.nvidia.com/cuda/cusparse/index.html#coordinate-coo>`_. Sparse
+  operations using matrices with the COO format without the property
+  ``matrix_property::sorted_by_rows`` or ``matrix_property::sorted`` will throw
+  an ``oneapi::mkl::unimplemented`` exception.
+- Using ``spmm`` with the algorithm ``spmm_alg::csr_alg3`` and an ``opA`` other
+  than ``transpose::nontrans`` or an ``opB`` ``transpose::conjtrans`` will throw
+  an ``oneapi::mkl::unimplemented`` exception.
```

Comment:
```suggestion
  a ``oneapi::mkl::unimplemented`` exception.
```

Reply:
I used `an` in multiple places like this because I don't think anyone reads the namespace. Do you still prefer it with `an`? I've also done this in the spec so I would also change it there if we go with this. I couldn't find a similar pattern in other domains oddly enough.

Reply:
Ok, this is changed in https://github.com/oneapi-src/oneMKL/pull/527/commits/00e5cedf62a20dbe016241b4aecfd8c3fd1b9479 and I also changed similar patterns in the spec as part of https://github.com/uxlfoundation/oneAPI-spec/pull/583

Reply:
Thanks for fixing that in the spec PR too.